apps/tasks/lib/pages/login_page.dart

```diff
@@ -33,9 +34,9 @@ class _LoginPageState extends State<LoginPage> {
           onPressed: () {
             if (_formKey.currentState!.validate()) {
               _formKey.currentState!.save();
-              // TODO do use email password here
+              // TODO do use email password here for login
```

Comment:
This should be clear now - maintainer meeting!

Reply:
seperate issue #192